src/input/target/mod.rs

```diff
@@ -59,7 +61,7 @@ pub mod xbox_series;
 /// Possible errors for a target device client
 #[derive(Error, Debug)]
 pub enum InputError {
-    #[error("error occurred running device")]
+    #[error("error occurred running input device")]
```

Comment:
Since you're touching this, could you update this to ensure we also print the downstream error message?:
```suggestion
    #[error("error occurred running input device: `{0}`")]
```

(Usage docs for `thiserror`: https://github.com/dtolnay/thiserror?tab=readme-ov-file#details)